cmd/packer-sdc/internal/mapstructure-to-hcl2/mapstructure-to-hcl2.go

```diff
@@ -52,6 +53,8 @@ var (
 	readme string
 )
 
+const HCLLABELINDEXKEY = "hcllabelindex"
```

Comment:
I'm not sure if this suggestion is idiomatic but I had trouble reading this variable name on my first time through (I kept reading it as Belinda Key 🤦‍♀️) I'd recommend naming it HCL_LABEL_INDEX_KEY